safeeyes/safeeyes/config/lang/tr.json

```diff
@@ -0,0 +1,46 @@
+{
+    "meta_info": {
+        "language_name": "Türkçe",
+        "language_name_en": "Turkish"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Biraz yürüyün",
+            "Arkanıza yaslanın ve biraz gevşeyin"
+        ],
+        "short_break_exercises": [
+            "Gözlerinizi sıkıca kapatın",
+            "Gözlerinizi çevirin",
+            "Gözlerinizi saat yönünde döndürün",
+            "Gözlerinizi saat yönünün tersine döndürün",
+            "Göz kapaklarınızı kapatıp açın",
+            "Uzak mesafede bir noktaya odaklanın",
+            "Gidip biraz su için"
+        ]
+    },
+    "messages": {
+        "ready_for_a_break": "{} Saniye içerisinde bir molaya hazır olun",
+        "safe_eyes_is_disabled": "Safe Eyes etkin değil",
```

Comment:
`safe_eyes_is_disabled` is replaced by:
`
"disabled_until_restart": "Disabled until restart",
 "disabled_until_x": "Disabled until {}",
`